tests/Feature/Api/V1/ConsoleIDsTest.php

```diff
@@ -32,18 +32,74 @@ public function testGetConsoleIds(): void
                 [
                     'ID' => $system1->ID,
                     'Name' => $system1->Name,
+                    'IconURL' => $system1->icon_url,
+                    'Active' => boolval($system1->active),
+                    'IsGameSystem' => System::isGameSystem($system1->ID),
                 ],
                 [
                     'ID' => $system2->ID,
                     'Name' => $system2->Name,
+                    'IconURL' => $system2->icon_url,
+                    'Active' => boolval($system2->active),
+                    'IsGameSystem' => System::isGameSystem($system2->ID),
                 ],
                 [
                     'ID' => $system3->ID,
                     'Name' => $system3->Name,
+                    'IconURL' => $system3->icon_url,
+                    'Active' => boolval($system3->active),
+                    'IsGameSystem' => System::isGameSystem($system3->ID),
                 ],
                 [
                     'ID' => $system4->ID,
                     'Name' => $system4->Name,
+                    'IconURL' => $system4->icon_url,
+                    'Active' => boolval($system4->active),
+                    'IsGameSystem' => System::isGameSystem($system4->ID),
+                ],
+            ]);
+    }
+
+    public function testGetConsoleIdsOnlyActive(): void
+    {
+        /** @var System $system1 */
+        $system1 = System::factory()->create(['active' => 1]);
+        /** @var System $system2 */
+        $system2 = System::factory()->create(['active' => 0]);
+
+        // only active systems
+        $this->get($this->apiUrl('GetConsoleIDs', ['a' => 1]))
+            ->assertSuccessful()
+            ->assertJsonCount(1)
+            ->assertJson([
+                [
+                    'ID' => $system1->ID,
+                    'Name' => $system1->Name,
+                    'IconURL' => $system1->icon_url,
+                    'Active' => boolval($system1->active),
+                    'IsGameSystem' => System::isGameSystem($system1->ID),
+                ],
+            ]);
+    }
+
+    public function testGetConsoleIdsOnlyGameSystems(): void
+    {
+        /** @var System $system1 */
+        $system1 = System::factory()->create();
+        /** @var System $system2 */
+        $system2 = System::factory()->create(['ID' => System::Events]);
+
+        // only game systems
+        $this->get($this->apiUrl('GetConsoleIDs', ['g' => 1]))
+            ->assertSuccessful()
+            ->assertJsonCount(1)
+            ->assertJson([
+                [
+                    'ID' => $system1->ID,
+                    'Name' => $system1->Name,
+                    'IconURL' => $system1->icon_url,
+                    'Active' => boolval($system1->active),
+                    'IsGameSystem' => System::isGameSystem($system1->ID),
```

Comment:
I think we should remove the `::isGameSystem()` calls from these `assertJson`s and explicitly check for `true` or `false`. This will cause a test failure if someone makes an actual behavior change.

Right now, if the Events system ID were to somehow change, tests would continue passing while API consumers might actually break. I think a test failure in that circumstance is desirable.